commands/operator-sdk/cmd/new.go

```diff
@@ -119,6 +122,8 @@ func mustBeNewProject() {
 }
 
 func doScaffold() {
+	log.Infof("Generating %s project scaffold.", strings.Title(operatorType))
```

Comment:
Do we need this and Line 153? The log on line 84 pretty much says this already including the type.